vizro-core/docs/pages/user-guides/custom-figures.md

```diff
@@ -104,7 +104,7 @@ For instance, to make a KPI card with the icon positioned on the right side of t
         1. Here we decorate our custom figure function with the `@capture("figure")` decorator.
         1. The custom figure function needs to have a `data_frame` argument and return a `Dash` component.
         1. We adjust the return statement to include the icon on the right side of the title. This is achieved by swapping the order of the `html.H2` and `html.P` compared to the original `kpi_card`.
-        1. This creates a [`layout`](layouts.md) with four rows and columns. The KPI cards are positioned in the first two cells, while the remaining cells are empty.
+        1. This creates a [`grid`](layouts.md) with four rows and columns. The KPI cards are positioned in the first two cells, while the remaining cells are empty.
```

Comment:
Would you be able to add the number of columns too, so it's super-clear to the reader? So "four rows and four columns" if that's the intent.